_includes/codepens/basic-example/index.js

```diff
@@ -9,10 +9,7 @@ tinymce.init({
   ],
   toolbar: 'undo redo | formatselect | ' +
```

Comment:
This also has duplicate spaces, but it was half okay/half bad before :woman_shrugging:

Reply:
fixed

Reply:
https://github.com/tinymce/tinymce-docs/pull/1426/commits/49f17aabc2e94afcff276b9e332dab05e39d67ae